src/jit/compiler.h

```diff
@@ -1298,7 +1298,7 @@ class fgArgInfo
 
     void SortArgs();
 
-    void EvalArgsToTemps();
```

Comment:
>EvalArgs [](start = 9, length = 8)

Could you change this to be in the positive sense? Perhaps `allowHfa`?

Reply:
Addressed